source/front-end/composants/screens/CompteResultat.svelte

```diff
@@ -0,0 +1,81 @@
+<script>
+  // @ts-check
+
+  import Skeleton from "../Skeleton.svelte";
+  import EtatDesComptes from "../../../format-données/produireEtatDesComptes";
+  import OpHautNiveauVersOpDeCompte from "../../../format-données/traduireOpérationsHautNiveauEnOpérationsDeCompte";
+  import { formatMontant, libelleCompte } from "../../stringifiers";
+
+  export let login;
+  export let logout;
+  export let ophn;
+
+  let data;
+  let annee;
+  let listeAnnees = [];
+
+  function getAnneesDispo(anneesDispo) {
+    let listeAnnees = [];
+    let annee = anneesDispo.next();
+    while (!annee.done) {
+      listeAnnees.push(annee.value);
+      annee = anneesDispo.next();
```

Comment:
C'est possible d'utiliser une boucle `for of` ici et ça serait plus clair et plus simple je pense (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Iterator#using_an_iterator_as_an_iterable)